Traceback (most recent call last):
  File "/home/mwess/.local/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/mwess/.local/lib/python3.10/site-packages/nbclient/client.py", line 1263, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/mwess/.local/lib/python3.10/site-packages/nbclient/util.py", line 85, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/mwess/.local/lib/python3.10/site-packages/nbclient/util.py", line 60, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/home/mwess/.local/lib/python3.10/site-packages/nbclient/client.py", line 701, in async_execute
    await self.async_execute_cell(
  File "/home/mwess/.local/lib/python3.10/site-packages/nbclient/client.py", line 1019, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/mwess/.local/lib/python3.10/site-packages/nbclient/client.py", line 913, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------

now = time()
nowstart = now

times = []
energies = []
tmpH = gfH.vec.CreateVector()
tmpE = gfE.vec.CreateVector()
with TaskManager():
    while t<tend:
        if i%drawevery == 0:
            gfH_history.AddMultiDimComponent(gfH.vec)
            #scene.Redraw()
            times.append(t)
            tmpH.data = massH * gfH.vec
            tmpE.data = massE * gfE.vec
            energies.append(InnerProduct(gfE.vec,tmpE)+InnerProduct(gfH.vec,tmpH))
            #print("\r time = {}\t step = {}\t energy = {}\t current dofs/s = {:e}".format(t,i,energies[-1],(fesE.ndof+fesH.ndof)*drawevery/(time()-now)),end="")
            now = time()
        i=i+1
        t+=tau
        gfH.vec.data += -tau*massinvH@Grad.T*gfE.vec
        gfE.vec.data += tau*massinvE@Grad*gfH.vec

comptime = time()-nowstart
print("\n timesteps: {}\t dofs: {}\t dofs per second: {:e}".format(tend/tau, (fesE.ndof+fesH.ndof),(fesE.ndof+fesH.ndof)*tend/tau/comptime))
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Input [0;32mIn [7][0m, in [0;36m<cell line: 6>[0;34m()[0m
[1;32m      4[0m times [38;5;241m=[39m []
[1;32m      5[0m energies [38;5;241m=[39m []
[0;32m----> 6[0m tmpH [38;5;241m=[39m [43mgfH[49m[38;5;241m.[39mvec[38;5;241m.[39mCreateVector()
[1;32m      7[0m tmpE [38;5;241m=[39m gfE[38;5;241m.[39mvec[38;5;241m.[39mCreateVector()
[1;32m      8[0m [38;5;28;01mwith[39;00m TaskManager():

[0;31mNameError[0m: name 'gfH' is not defined
NameError: name 'gfH' is not defined

